Platform/ElkhartlakeBoardPkg/Library/Stage2BoardInitLib/Stage2BoardInitLib.c

```diff
@@ -526,6 +526,31 @@ InitializeSmbiosInfo (
   //
   // SMBIOS_TYPE_PROCESSOR_INFORMATION : TBD
   //
+  //
+  // SMBIOS_TYPE_PHYSICAL_MEMORY_ARRAY
+  //
+  AddSmbiosTypeString (&TempSmbiosStrTbl[Index++], SMBIOS_TYPE_PHYSICAL_MEMORY_ARRAY,
+    1, "2");
+
+  //
+  // SMBIOS_TYPE_MEMORY_DEVICE
+  //
+  AddSmbiosTypeString (&TempSmbiosStrTbl[Index++], SMBIOS_TYPE_MEMORY_DEVICE,
+    1, "Controller0-ChannelA");
+  AddSmbiosTypeString (&TempSmbiosStrTbl[Index++], SMBIOS_TYPE_MEMORY_DEVICE,
+    2, "BANK 0");
+  AddSmbiosTypeString (&TempSmbiosStrTbl[Index++], SMBIOS_TYPE_MEMORY_DEVICE,
+    3, "Micron Technology");
+  AddSmbiosTypeString (&TempSmbiosStrTbl[Index++], SMBIOS_TYPE_MEMORY_DEVICE,
+    4, "20000000");
+  AddSmbiosTypeString (&TempSmbiosStrTbl[Index++], SMBIOS_TYPE_MEMORY_DEVICE,
+    5, "9876543210");
```

Comment:
MEMORY_ASSET_TAG is defined, should it be used here instead of using the hardcoded strings?

Reply:
eliminated hardcoded since it has been defined in MEMORY_ASSET_TAG, but instead here, the MEMORY_ASSET_TAG is use in SmbiosInitLib.c